db/seeds.rb

```diff
@@ -4,36 +4,42 @@
 Dir[Rails.root.join('spec/factories/*.rb')].each { |f| require f }
 
 def create_sarce_trainer_minimal_stack
+  create_admin
   create_users
   create_documentations
   create_questions
   create_daily_questions
   create_given_answers
 end
 
+def create_admin
+  FactoryGirl.create(:user, :admin)
+  puts '- 1 admin created'
+end
+
 def create_users
   FactoryGirl.create_list(:user, 2)
   puts '- 2 users created'
 end
 
 def create_documentations
-  FactoryGirl.create_list(:documentation, 5)
-  puts '- 5 documentations created'
+  FactoryGirl.create_list(:documentation, 2)
+  puts '- 2 documentations created'
 end
 
 def create_questions
-  FactoryGirl.create_list(:question, 30)
-  puts '- 30 questions created'
+  FactoryGirl.create_list(:question, 5)
+  puts '- 5 questions created'
 end
 
 def create_daily_questions
-  FactoryGirl.create_list(:daily_question, 30)
-  puts '- 30 daily questions created'
+  FactoryGirl.create_list(:daily_question, 5)
+  puts '- 5 daily questions created'
 end
 
 def create_given_answers
```

Comment:
It would be better to be able to create a question + its given answer at the same time.
Without that, we would have questions without given answer on a side, and on the other side, a lonely question with 5 given answers.